docs/core/porting/winforms.md

```diff
@@ -30,7 +30,7 @@ In this article, various names are used to identify types of files used for migr
 - A working Windows Forms project in a solution that builds and runs without issue.
 - A project coded in C#.
 
-> [!NOTE]
+> [!NOTE] <!-- TODO: (Need guidance on how this note should be changed) -->
```

Comment:
@OliaG, Any recommendation is this note?
It was added from a blog of you when version 16.5 was still in preview, that's outdated now.
Should the note be completely removed? or you do have another recommendation?

Reply:
No I believe this remains, I checked the latest preview and the option is still listed under preview features, so I think it's good to stay for now. However, we can update references from core 3.0 to core 3.1